vignettes/hermes.Rmd

```diff
@@ -17,13 +15,16 @@ author:
   - Chendi Liao
   - Carolyn Zhang
   - Joseph N. Paulson
+  - Alex Richardson
+  - name: Daniel Sabanés Bové
+    email: daniel.sabanes_bove@roche.com
 output: 
   BiocStyle::html_document:
     toc_float: true
 vignette: |
   %\VignetteIndexEntry{Introduction to `hermes`}
-  %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
 editor_options:
   chunk_output_type: console
 ---
```

Comment:
can you still add here the `draw_heatmap()` explanations in the right subsection?

Reply:
Yes see the update, let me know if that looks right or what needs to change.

Reply:
thanks update looks good